src/status_im2/contexts/wallet/account/tabs/about/view.cljs

```diff
@@ -6,23 +6,26 @@
     [status-im2.contexts.wallet.common.temp :as temp]
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]
-    [status-im2.contexts.wallet.account.tabs.about.style :as style]))
+    [status-im2.contexts.wallet.account.tabs.about.style :as style]
+    [quo2.theme :as quo.theme]))
 
 (defn description
-  [{:keys [address]}]
-  [quo/text {:size :paragraph-2}
-   (map (fn [network]
-          ^{:key (str network)}
-          [quo/text
-           {:size   :paragraph-2
-            :weight :medium
-            :style  {:color (colors/custom-color network)}}
-           (str (subs (name network) 0 3) (when (= network :arbitrum) "1") ":")])
-        temp/network-names)
-   [quo/text
-    {:size   :paragraph-2
-     :weight :monospace}
-    address]])
+  [{:keys [address theme]}]
+  (let [networks-list (rf/sub [:wallet-2/network-details])]
+    [quo/text {:size :paragraph-2}
+     (map (fn [{:keys [short-name network-name]}]
+            (prn network-name)
+            ^{:key (str name)}
```

Comment:
Can we use the chain ID as the key here as they are unique?

Reply:
Sure 👌